### ISSUE TITLE
Load data through GPUDirect Storage and/or RTX I/O

### ISSUE THREAD

**Issue by kleingeo** [dask/dask#6712]
When loading a numpy or image, is the API capable of accelerating this through GPUDirect Storage and/or RTX I/O? If not, are there plans on adding this support?

**Comment by mrocklin** [comment]
cc @jakirkham @quasiben

**Comment by jakirkham** [comment]
I don't think this would be something Dask itself would do (could be wrong about this), but likely through a 3rd party library it could be possible.

**Comment by jakirkham** [comment]
How do you handle this without Dask today?
